### PR TITLE
Top

### PR DESCRIPTION
**Pull request by glasnt**
Puns, Vicky! Puns!

### CODE REVIEW

File: top/toptips.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  193 root      20   0  123520  29636   8640 S   0.0  0.7   0:00.58 flask
+  195 root      20   0   23608   2724   2400 R   0.0  0.1   0:00.21 top
+```
+
+_Your version of `top` may differ from this, particularly in the columns that are displayed._
+
+## How to read the output
+
+We can tell what we're running based on this output, but trying to interpret the output can be slightly confusing. 
+
```
Comment (vmbrasseur) [inline top/toptips.md]: "output" twice in rapid succession. A-OK to replace second one with "it". Yay, antecedents.

File: top/toptips.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+At the top we can see a bunch of statistics, and then we can see a list of things.
```
Comment (vmbrasseur) [inline top/toptips.md]: "In the first line" instead of "at the top"? Probably best to avoid too many usages of the word 'top' in a single article. ;-)

Comment (vmbrasseur) [reply top/toptips.md]: Also, expand on "a list of things" a bit to help transition into the next section. Maybe something like, "we see a list of things that are a bit harder to understand" or some such?

File: top/toptips.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+## How to read the output
+
+We can tell what we're running based on this output, but trying to interpret the output can be slightly confusing. 
+
+
+At the top we can see a bunch of statistics, and then we can see a list of things. 
+
+### The list
+
+These are the *processes* that are running on the system which are ordered, by default, by CPU usage, in descending order.
```
Comment (vmbrasseur) [inline top/toptips.md]: To make it a bit more clear for the new folks, maybe clarify further with, "This means the processes that are having the highest impact on your CPU will be at the top of the list." ?

File: top/toptips.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+ - `SHR`: Shared Memory Size in KiB (kibibytes)\* - the "shared memory", a subset of VIRT
+ - `S`: Process State - one of: "I"dle, "R"unning, "S"leeping, "Z"ombie, Stopped ("T" or "t"), and other some less common options.
+ - `%CPU`: Percentage of CPU Usage since the last screen update
+ - `%MEM`: Percentage of Memory Usage (RES) since the last screen update
+ - `TIME+`: Total CPU time used since the process started. 
+ - `COMMAND`: the command
+
+A few notes on `VIRT`, `RES`, and `SHR`: 
+
+The exact mappings of what these three values represent is something which in everyday operations doesn't *really* matter. The most imporatnt thing is that the process with the most `VIRT` the process using the most memory. If you're in `top` because you're debugging why your computer feels like it's in a pool of molasses, the process with the biggest VIRT number is the culprit. If you would like to learn exactly what the "shared" and "physical" memory refers to, check out the "Linux Memory Types" of the [`top` manual](http://man7.org/linux/man-pages/man1/top.1.html)
```
Comment (vmbrasseur) [inline top/toptips.md]: Missing an 'is' after "VIRT".

File: top/toptips.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+(And yes, I did type ki*bi*bytes, not ki*lo*bytes. The 1024 value that you normally refer to as a kilobyte is *actually* a kibibyte. The Greek *kilo* ("χίλιοι") means *thousand*, and is used to mean 1,000 of something (a kilometer -- a thousand meters, a kilogram -- a thousand grams). Kibi is a portmanteau of *kilo* and *byte*, and means 1024 bytes (or 2 to the power of 10, 2^10). But, because words are hard to say, many people say *kiloybyte* when they mean 1024 bytes. All of this to say that `top` is trying to use the proper terms here, so just go with it. #themoreyouknow)
+
+A note on screen updates: 
+
+One of the objectionably __really cool things__ that linux programs can do is update their own display in a window so that they are updated live, and appear animated. In our case, the time between updates is important, because some of our statistics (`%CPU`, `%MEM`) are based on the value since the last screen update.
```
Comment (vmbrasseur) [inline top/toptips.md]: Objectionably? I suspect your brain mistyped "objectively". :-)

Comment (vmbrasseur) [reply top/toptips.md]: Might wanna do a find/replace to capitalise Linux everywhere.

File: top/toptips.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+ - `load average` (then three numbers): The [system load](https://en.wikipedia.org/wiki/Load_(computing)) of over the last 1, 5, and 15 minutes, respectively
+
+The second row shows task information, and it's fairly self explanatory: it shows the total number of processes, and the number of running, sleeping, stopped, and zombie processes. This is literally a sum of the `S` (State) column from earlier.
+
+ 
+The third row: CPU information. This shows the CPU usage, separated by types. This data is the value between screen refreshes. The values are: 
+
+ - `us`: user processes
+ - `sy`: system processes
+ - `ni`: nice user processes
```
Comment (vmbrasseur) [inline top/toptips.md]: Maybe link to a `nice` article or something?

**Comment by vmbrasseur** [comment]
Comments left. Only minimal changes needed. Wooooo!

**Comment by glasnt** [comment]
Thanks @vmbrasseur. Updates made. 

I'll leave this bake for a day then I'll take another look over it, and let you know when I think it's good to go! ✨

**Pull request merged**